cloud_blobstore/__init__.py

```diff
@@ -117,7 +117,9 @@ def upload_file_handle(
             self,
             bucket: str,
             key: str,
-            src_file_handle: typing.BinaryIO):
+            src_file_handle: typing.BinaryIO,
+            content_type: str = None,
```

Comment:
PEP calls for no spaces around `=` when used as a default value.

Reply:
fill fix